### DISCUSSION TITLE
SB5.1 - OCI - Deploy/Kubernetes/Analytics failing

### DISCUSSION THREAD

**Discussion by ddevadat** [Sunbird-Ed/Community#326] [Installation] [answered]
CSP: OCI
release: release-5.0.0_RC2

The deployment Deploy/Kubernetes/Analytics is failing. The init container keeps crashing

`
jenkins@jenkins:~$ k logs -f analytics-5bbcd9445-96tl5 -c proxy-init
iptables v1.6.0: can't initialize iptables table `nat': Table does not exist (do you need to insmod?)
Perhaps iptables or your kernel needs to be upgraded. 
`

**Comment by ddevadat** [comment]
Init Containers:
  proxy-init:
    Container ID:  cri-o://0ec5790ba97492356119dfd0552c65767965bad964fec414e25a2e9809d84ef3
    Image:         openpolicyagent/proxy_init:v5
    Image ID:      docker.io/openpolicyagent/proxy_init@sha256:8f988a690516728b08956f2838a7a08202547e44d2c962009301d738c7a4abae
    Port:          <none>
    Host Port:     <none>
    Args:
      -p
      9999
      -u
      1111
      -w
      8282,10000
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    3
      Started:      Thu, 29 Dec 2022 15:14:45 +0530
      Finished:     Thu, 29 Dec 2022 15:14:45 +0530
    Ready:          False
    Restart Count:  2

**Comment by ddevadat** [comment, marked as answer]
I think this is affecting all deployment which is having OPA

**Reply by beepdot** [reply]
Please disable opa for now. We will add the docs on how to configure opa post which we can enable them. These are the vars you can define in private repo and set it to `false` to disable opa.
https://github.com/project-sunbird/sunbird-devops/blob/release-5.1.0/ansible/roles/stack-sunbird/defaults/main.yml#L1035-L1042
